Library/Formula/syncthing-inotify.rb

```diff
@@ -0,0 +1,63 @@
+class SyncthingInotify < Formula
+  desc "File watcher intended for use with Syncthing"
+  homepage "https://github.com/syncthing/syncthing-inotify"
+  url "https://github.com/syncthing/syncthing-inotify.git",
+    :tag => "v0.6.5", :revision => "2b290b929e18e7cd1cfea11240e4fa870d6638d5"
```

Comment:
Why not using a tarball? https://github.com/syncthing/syncthing-inotify/archive/v0.6.5.tar.gz